Document: npm install requires python2 dependency

npm install fails unless you have python2 installed. Some operating systems (e.g., latest Fedora Silverblue do not come with Python 2).

This should be documented in the readme.

Also just tried in a toolbox (container) that has Python 2 and now getting gyp errors (tried with Node 19.5.0 and 18.14.0 LTS):

```
npm ERR! gyp verb common.gypi checking for gypi file: /var/home/aral/Projects/other/htmx/node_modules/node-sass/common.gypi
npm ERR! gyp verb gyp gyp format was not specified; forcing "make"
npm ERR! gyp info spawn /usr/bin/python2
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/var/home/aral/Projects/other/htmx/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/var/home/aral/Projects/other/htmx/node_modules/node-sass/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/var/home/aral/Projects/other/htmx/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/var/home/aral/.node-gyp/19.6.0/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/var/home/aral/.node-gyp/19.6.0',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/var/home/aral/Projects/other/htmx/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/var/home/aral/.node-gyp/19.6.0/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/var/home/aral/Projects/other/htmx/node_modules/node-sass',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp: Undefined variable standalone_static_library in binding.gyp while trying to load binding.gyp
npm ERR! gyp ERR! configure error 
npm ERR! gyp ERR! stack Error: `gyp` failed with exit code: 1
npm ERR! gyp ERR! stack     at ChildProcess.onCpExit (/var/home/aral/Projects/other/htmx/node_modules/node-gyp/lib/configure.js:345:16)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:512:28)
npm ERR! gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:293:12)
npm ERR! gyp ERR! System Linux 6.1.7-200.fc37.x86_64
npm ERR! gyp ERR! command "/var/home/aral/.local/share/nvm/v19.6.0/bin/node" "/var/home/aral/Projects/other/htmx/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
npm ERR! gyp ERR! cwd /var/home/aral/Projects/other/htmx/node_modules/node-sass
npm ERR! gyp ERR! node -v v19.6.0
npm ERR! gyp ERR! node-gyp -v v3.8.0
npm ERR! gyp ERR! not ok 
npm ERR! Build failed with error code: 1

npm ERR! A complete log of this run can be found in:
npm ERR!     /var/home/aral/.npm/_logs/2023-02-08T14_15_59_620Z-debug-0.log
```

Looks like node-sass is rather problematic.

Will continue trying to get started… :)

--- a/README.md
+++ b/README.md
@@ -7,7 +7,6 @@
 [![Bundlephobia](https://badgen.net/bundlephobia/dependency-count/htmx.org)](https://bundlephobia.com/result?p=htmx.org)
 [![Bundlephobia](https://badgen.net/bundlephobia/minzip/htmx.org)](https://bundlephobia.com/result?p=htmx.org)
 
-
 ## introduction
 
 htmx allows you to access  [AJAX](https://htmx.org/docs#ajax), [CSS Transitions](https://htmx.org/docs#css_transitions),
@@ -67,20 +66,24 @@ Note there is an old broken package called `htmx`.  This is `htmx.org`.
 * please write code, including tests, in ES5 for [IE 11 compatibility](https://stackoverflow.com/questions/39902809/support-for-es6-in-internet-explorer-11)
 * please include test cases in [`/test`](https://github.com/bigskysoftware/htmx/tree/dev/test) and docs in [`/www`](https://github.com/bigskysoftware/htmx/tree/dev/www)
 * if you are adding a feature, consider doing it as an [extension](https://htmx.org/extensions) instead to
-keep the core htmx code tidy
+  keep the core htmx code tidy
 * development pull requests should be against the `dev` branch, docs fixes can be made directly against `master`
 * No time? Then [become a sponsor](https://github.com/sponsors/bigskysoftware#sponsors)
 
 ### hacking guide
 
 To develop htmx locally, you will need to install the development dependencies.
-Use node 15 and run:
+
+__Requires Python 2 and Node 15.__
+
+Run:
 
 ```
 npm install
 ```
 
 Then, run a web server in the root.
+
 This is easiest with Python:
 
 ```

